recipes-core/cvm-reverse-proxy-client/cvm-reverse-proxy-client_v0.1.0.bb

```diff
@@ -6,7 +6,7 @@ LIC_FILES_CHKSUM = "file://src/${GO_WORKDIR}/LICENSE;md5=4ae09d45eac4aa08d013b5f
 inherit go-mod update-rc.d
 
 INITSCRIPT_NAME = "cvm-reverse-proxy-client-init"
-INITSCRIPT_PARAMS = "defaults 87"
+INITSCRIPT_PARAMS = "defaults 86"
```

Comment:
This is now on the same level of cvm-reverse-proxy-server. I had them before running on the same level because they are not dependent on each other. However, I noticed that only one of them writes to the fifo file and the other is for some reason gets suppressed. That is why I put them on different sequential run levels. 
Maybe you need to test this on a test image and see if all services logging correctly and nothing is missing in the events

Reply:
noted. Will test once everything is ready.